components/brave_wallet/browser/swap_response_parser.cc

```diff
@@ -21,6 +25,50 @@ bool ParseResultFromDict(const base::DictionaryValue* response_dict,
   return true;
 }
 
+bool ParseUint64ResultFromValue(const base::Value& value,
+                                const std::string& key,
+                                uint64_t* output_val) {
+  auto val = value.FindIntKey(key);
```

Comment:
Not safe, shouldn't use int for uint64_t values.

Reply:
Fixed here e918beffe639e2f65fd97084ce58e05f75722925.